jax/_src/pallas/triton/lowering.py

```diff
@@ -588,12 +590,26 @@ def inner(ctx: LoweringRuleContext, *args: ir.Value) -> ir.Value:
 
 _abs_dispatch_table = _make_dispatch_table(
     "abs",
-    [
-        _Extern(["int32"], "__nv_abs", "int32"),
-        _Extern(["int64"], "__nv_llabs", "int64"),
-        _Extern(["float32"], "__nv_fabsf", "float32"),
-        _Extern(["float64"], "__nv_fabs", "float64"),
-    ],
+      { "cuda":
+        [
+          _Extern(["int32"], "__nv_abs", "int32"),
+          _Extern(["int64"], "__nv_llabs", "int64"),
+          _Extern(["float32"], "__nv_fabs_f", "float32"),
+          _Extern(["float64"], "__nv_fabs", "float64"),
+        ],
+        "rocm":
+        [
+          #\todo: rahbatra - add support for hip device functions
```

Comment:
Can you remove the TODO with a commented out code, please?

Reply:
Done!